robottelo/ui/locators/base.py

```diff
@@ -943,6 +944,10 @@
                           "//input[@placeholder='Filter permissions']"),
     "roles.perm_type": (By.XPATH, "//label[contains(., '%s')]"),
     "roles.permission": (By.XPATH, "//input[@value='%s']"),
+    "roles.resources": (
+        By.XPATH, "//table[contains(@id, 'DataTables_Table')]/tbody/tr/td[1]"),
+    "roles.permissions": (
+        By.XPATH, "//td[contains(text(), '%s')]/following-sibling::td[1]"),
```

Comment:
Same here, `//table[contains(@id, 'DataTables')]/tbody/tr/td[contains(text(), '%s')]/following-sibling::td[count(//table/thead/tr/th[.="Permissions"]/preceding-sibling::th)]`
Though here idea was to fetch permissions by one row using resource type name as filter, so only one element would be returned.